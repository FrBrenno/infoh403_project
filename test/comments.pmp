'' This input tries to deal with comments ''

begin
    checkpoint1
    '' this comment should be ignore ''

    checkpoint2
    ** this comment should be ignore **

    checkpoint3
    ** a := b + 1

    checkpoint4
    ** is this ignored ** i do not know

    checkpoint5
    '' For some reason this comment does not work
    ''

    checkpoint6
    '' a nested comment **would work? ''

    checkpoint7
    '' and now ** '' ** ?''

    checkpoint8
    ** '' this is a long comment in a short comment''
end