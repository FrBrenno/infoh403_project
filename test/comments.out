token: begin	lexical unit: BEG
token: checkpoint1	lexical unit: VARNAME
token: checkpoint2	lexical unit: VARNAME
token: checkpoint3	lexical unit: VARNAME
token: checkpoint4	lexical unit: VARNAME
token: checkpoint5	lexical unit: VARNAME
token: checkpoint6	lexical unit: VARNAME
token: checkpoint7	lexical unit: VARNAME
token: checkpoint8	lexical unit: VARNAME
token: end	lexical unit: END

Variables
checkpoint8 26
checkpoint2 7
checkpoint3 10
checkpoint1 4
checkpoint6 20
checkpoint7 23
checkpoint4 13
checkpoint5 16
